README.md

```diff
@@ -29,15 +29,15 @@ scroll-boundary-behavior{-x,-y}: auto | contain | none
 
 where:
 * auto - Allow the default behavior for the user agent.
-* contain - Hint to disable scroll chaining. The user agent may show an appropriate overscroll affordance. If the scroll chaining would trigger a non-scroll action, such as a navigation action, this property should be a hint to disable it.
+* contain - Hint to disable scroll chaining. The user agent may show an appropriate overscroll affordance such as a glow/bounce etc. If the scroll chaining would trigger a non-scroll action, such as a navigation action, this property should be a hint to disable the navigation action.
```

Comment:
I think it's best not to mention glow/bounce here since they are not part of any specification. I left it out intentionally. The spec could maybe still include it under a non-normative example section. The rest looks great.